references/cli/addon.go

```diff
@@ -1002,7 +1002,7 @@ func waitApplicationRunning(k8sClient client.Client, addonName string) error {
 		case common2.ApplicationWorkflowSuspending:
 			fmt.Printf("Enabling suspend, please run \"vela workflow resume %s -n vela-system\" to continue", addonutil.Addon2AppName(addonName))
 			return nil
-		case common2.ApplicationWorkflowTerminated:
+		case common2.ApplicationWorkflowTerminated, common2.ApplicationWorkflowFailed:
```

Comment:
seems we don't have terminated state now?  \cc @FogDong

Reply:
We still have the termianted state if the user termiante the workflow manually.

Reply:
For early versions like v1.5, if the workflow step is failed after retries, the workflow's state is terminated. We add failed state in v1.6.